feature_apps/src/main/java/cm/aptoide/pt/feature_apps/presentation/AppsView.kt

```diff
@@ -1,20 +1,127 @@
 package cm.aptoide.pt.feature_apps.presentation
 
-import androidx.compose.foundation.layout.Column
-import androidx.compose.foundation.layout.fillMaxSize
-import androidx.compose.foundation.layout.wrapContentSize
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.foundation.lazy.LazyRow
+import androidx.compose.foundation.lazy.items
+import androidx.compose.material.CircularProgressIndicator
+import androidx.compose.material.MaterialTheme
 import androidx.compose.material.Text
 import androidx.compose.runtime.Composable
+import androidx.compose.runtime.collectAsState
+import androidx.compose.runtime.getValue
 import androidx.compose.ui.Alignment
 import androidx.compose.ui.Modifier
 import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import cm.aptoide.pt.feature_apps.data.App
+import cm.aptoide.pt.feature_apps.domain.Bundle
+import cm.aptoide.pt.feature_apps.domain.Type
+import java.util.*
+
+@Composable
+fun AppsScreen(viewModel: BundlesViewModel, type: ScreenType) {
+  val bundles: List<Bundle> by viewModel.bundlesList.collectAsState(initial = emptyList())
+  val isLoading: Boolean by viewModel.isLoading
+  BundlesScreen(isLoading, bundles)
+}
 
-@Preview
 @Composable
-fun AppsScreen() {
-  Column(modifier = Modifier
+private fun BundlesScreen(
+  isLoading: Boolean,
+  bundles: List<Bundle>,
+) {
+  Column(
+    modifier = Modifier
       .fillMaxSize()
-      .wrapContentSize(Alignment.Center)) {
-    Text(text = "Apps")
+      .wrapContentSize(Alignment.TopCenter)
+  ) {
+    if (isLoading)
+      CircularProgressIndicator()
+    else
+      LazyColumn(modifier = Modifier
+        .fillMaxSize()
+//        .verticalScroll(rememberScrollState())   Error: Nesting scrollable in the same direction layouts like LazyColumn and Column(Modifier.verticalScroll())
```

Comment:
comments here and bellow, but i guess is just a case to be handled later, so if  it is, just ignore.